extensions/multiline.md

```diff
@@ -80,7 +80,7 @@ Servers MUST NOT reject blank lines other than in the following cases:
 * Clients MUST NOT send blank lines with the `draft/multiline-concat` tag.
 * Clients MUST NOT send messages consisting entirely of blank lines.
 
-Clients MUST NOT send messages other than PRIVMSG while a multiline batch is open.
+Clients MUST NOT send messages other than PRIVMSG or NOTICE while a multiline batch is open.
```

Comment:
I get the intent, but formally, this probably needs to be reworded to disallow NOTICE in a PRIVMSG batch and vice-versa

Reply:
I believe this is covered by the first change: "Multiline batches MUST only contain one or more PRIVMSG lines, or one or more NOTICE lines."

Reply:
Hmm, if I understand this line correctly, its purpose is to disallow messages that don't belong to the batch. But it fails to do so (even with PRIVMSG only), because it doesn't disallow this:

```

    Client: BATCH +792da7 draft/multiline #channel
    Client: @batch=792da7 PRIVMSG #channel :message1
    Client: PRIVMSG #channel :message2
    Client: BATCH -792da7
```

Unless it's intentional to allow PRIVMSGs that don't belong to the batch? If so, NOTICE should also be allowed, independently of whether this PR is accepted or not.